webpack.config.js

```diff
@@ -262,6 +263,13 @@ module.exports = {
                 exclude: path.resolve(__dirname, 'node_modules/font-awesome/'),
                 loader: 'file-loader',
                 options: { name: 'img/[name].[md5:hash:base64:6].[ext]' }
+            },
+            {
+                test: /\.(ogg|mp3|wav|mpe?g)$/i,
```

Comment:
Do all of the file types work? If so, lets add a message somewhere noting the valid extensions, if not lets trim this down to just valid.

Reply:
@ftwpala

Reply:
@jbpratt78 oh right, sec